### PR TITLE
PTOCP-947: Client generated code for upgrade policy

### LINKED ISSUE #42: Add Schedule Software/Config Updates support
Could you add support for the BloxOne /api/upgrade_policy/ APIs, so we can manage maintenance windows and config updates via Go? The OpenAPI documentation can be found here: https://csp.infoblox.com/apidoc/docs/UpgradePolicy.

### PR DESCRIPTION
**Pull request by shrutipathak2**
Add Schedule Software/Config Updates support 
PTOCP-947: Client generated code for upgrade policy

### CODE REVIEW

**Comment by mathewab** [comment]
looks like you have both models with and without prefix -  `model_service_v2_...` and also `model_...` 

Can you clean the entire directory and regenerate? Also, please run `make fmt` after generation.

File: upgradepolicy/docs/UpgradePolicyV2API.md [hunk 1]
```diff
@@ -0,0 +1,473 @@
+# UpgradePolicyV2API
+
+All URIs are relative to *http://localhost/api/upgrade_policy*
```
Comment (mathewab) [inline upgradepolicy/docs/UpgradePolicyV2API.md]: This should be csp.infoblox.com

In the generator, the swagger.json might be missing the following line, can you please manually add it and regenerate? 
  "host": "csp.infoblox.com"

**Comment by mathewab** [comment]
Please add the API to the project README (https://github.com/infobloxopen/bloxone-go-client/blob/master/README.md) and aggregated client (https://github.com/infobloxopen/bloxone-go-client/blob/master/client/client.go) as well

File: upgradepolicy/docs/UpgradePolicyV2API.md (hunk 1 above)
Comment (shrutipathak2) [reply upgradepolicy/docs/UpgradePolicyV2API.md]: Done

**Comment by shrutipathak2** [comment]
> looks like you have both models with and without prefix - `model_service_v2_...` and also `model_...`
> 
> Can you clean the entire directory and regenerate? Also, please run `make fmt` after generation.

Done

**Comment by shrutipathak2** [comment]
> Please add the API to the project README (https://github.com/infobloxopen/bloxone-go-client/blob/master/README.md) and aggregated client (https://github.com/infobloxopen/bloxone-go-client/blob/master/client/client.go) as well

Done

File: README.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ The following Bloxone APIs are supported:
 - [DNS Configuration](dns_config/README.md)
 - [DNS Data](dns_data/README.md)
 - [Keys](keys/README.md)
+- [Upgrade Policy](upgradePolicy/README.md)
```
Comment (mathewab) [inline README.md]: can you move this to be under `## Bloxone Cloud`

Comment (shrutipathak2) [reply README.md]: Done

**Review by mathewab — APPROVED** [review]

**Pull request merged**